src/ControlSystem/TimescaleTuner.hpp

```diff
@@ -7,6 +7,10 @@
 
 #include "DataStructures/DataVector.hpp"
 
+namespace PUP {
+  class er;
+}
```

Comment:
Wrap in `\cond`.  Clang-tidy or something will also complain about the missing `// namespace PUP` at the end.